public/robots.txt

```diff
@@ -1,4 +1,4 @@
-# See http://www.robotstxt.org/wc/norobots.html for documentation on how to use the robots.txt file
+# See https://www.robotstxt.org/ for documentation on how to use the robots.txt file
```

Comment:
Should this remain `http://`? They seem to issue a redirect back on the https port (which seems like a curious thing to bother setting up, but whatever...).